src/main/java/com/telekom/javaschool/medicalrehapp/entity/Event.java

```diff
@@ -1,63 +1,41 @@
 package com.telekom.javaschool.medicalrehapp.entity;
 
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.Enumerated;
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
 import javax.persistence.OneToOne;
+import javax.persistence.Table;
 import java.time.LocalDateTime;
 
+@Getter
+@Setter
+@NoArgsConstructor
 @Entity
+@Table(name = "event")
 public class Event extends AbstractEntity {
 
     @ManyToOne
     @JoinColumn(name = "patient_id")
     private Patient patient;
+
+    @Column(name = "date_time")
     private LocalDateTime dateTime;
+
+    @Enumerated
```

Comment:
If the enumerated type is not specified, the EnumType value is assumed to be ORDINAL, but we have decided to store enums in the DB as a STRING.